src/quacc/calculators/mrcc/mrcc.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import annotations
+
+import re
+
+from ase.calculators.genericfileio import (
+    BaseProfile,
+    CalculatorTemplate,
+    GenericFileIOCalculator,
+)
+
+import quacc.calculators.mrcc.io as io
+
+
+def get_version_from_mrcc_header(mrcc_header):
+    match = re.search(r"Release date: (.*)$", mrcc_header, re.M)
+    return match.group(1)
+
+
+class MrccProfile(BaseProfile):
+    def __init__(self, binary, **kwargs):
+        """
+        Parameters
+        ----------
+        binary : str
+            Full path to the dmrcc binary, the path to the dmrcc_mpi binary must be specified if MRCC is to be run in parallel.
+        """
+        # Because MRCC handles its parallelization without being called with
+        # mpirun/mpiexec/etc parallel should be set to False.
+        # Whether or not it is run in parallel is controlled by mrccinput or mrccblocks
+        super().__init__(parallel=False, parallel_info={})
+        self.binary = binary
```

Comment:
Please note that ASE has updated its approach to the profiles, and `binary` is no longer present. Instead, `BaseProfile` has a `command` keyword argument. There is also no more `parallel` or `parallel_info` keyword argument.